tests/aequilibrae/project/test_project_triggers.py

```diff
@@ -0,0 +1,191 @@
+from unittest import TestCase
+import tempfile
+import os
+from warnings import warn
+from shutil import copytree, rmtree
+from aequilibrae.project import Project
+import uuid
+import sqlite3
+from ...data import no_triggers_project
+
+
+class TestProject(TestCase):
+    def setUp(self) -> None:
+        self.temp_proj_folder = os.path.join(tempfile.gettempdir(), uuid.uuid4().hex)
+        copytree(no_triggers_project, self.temp_proj_folder)
+        self.proj = Project()
+        self.proj.open(self.temp_proj_folder)
+        self.curr = self.proj.conn.cursor()
+
+    def tearDown(self) -> None:
+        self.proj.close()
+        rmtree(self.temp_proj_folder)
+
+    def test_link_type_triggers(self):
+        root = os.path.dirname(os.path.realpath(__file__)).replace('tests', '')
+        qry_file = os.path.join(root, "network/database_triggers/link_type_table_triggers.sql")
+        with open(qry_file, "r") as sql_file:
+            query_list = sql_file.read()
+            query_list = [cmd for cmd in query_list.split("#")]
+
+            def reboot_cursor():
+                self.proj.conn.commit()
+                self.curr = self.proj.conn.cursor()
```

Comment:
this shouldn't be nested in the with block

Reply:
I refactored this completely already. It is in a different pull request